kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/network/model/Offer.java

```diff
@@ -93,7 +92,9 @@ public OfferTypeEnum read(final JsonReader jsonReader) throws IOException {
     public enum ContentTypeEnum {
 
         POLL("poll"),
-        COUPON("coupon");
+        COUPON("coupon"),
+        EXTERNAL("external");
```

Comment:
does EXTERNAL should be part of network Offer object? - EXTERNAL is relevant in network order context not Offer as far as I know

Reply:
so how you would approach to deal with it?